components/dashboard/Menu.js

```diff
@@ -0,0 +1,373 @@
+        <React.Fragment>
+          <MenuLink isSub collective={collective} section={ABOUT_ORG_SECTIONS.INFO} />
+          <MenuLink isSub collective={collective} section={ABOUT_ORG_SECTIONS.COLLECTIVE_PAGE} />
+          <MenuLink isSub collective={collective} section={ABOUT_ORG_SECTIONS.CONNECTED_ACCOUNTS} />
+          <MenuLink isSub collective={collective} section={ABOUT_ORG_SECTIONS.TEAM} />
+          <MenuLink isSub collective={collective} section={ORG_BUDGET_SECTIONS.PAYMENT_METHODS} />
+        </React.Fragment>
+      )}
+      <MenuLink isSub collective={collective} section={ORG_BUDGET_SECTIONS.PAYMENT_RECEIPTS} />
+      {!isAccountantOnly && (
+        <React.Fragment>
+          <MenuLink isSub collective={collective} section={ORG_BUDGET_SECTIONS.GIFT_CARDS} />
+          <MenuLink isSub collective={collective} section={ALL_SECTIONS.WEBHOOKS} />
+          <MenuLink isSub collective={collective} section={COLLECTIVE_SECTIONS.FOR_DEVELOPERS} />
+          <MenuLink isSub collective={collective} section={COLLECTIVE_SECTIONS.ACTIVITY_LOG} />
+          <MenuLink isSub collective={collective} section={FISCAL_HOST_SECTIONS.SECURITY} />
+          <MenuLink isSub collective={collective} section={ALL_SECTIONS.ADVANCED} />
+          {!isHostAccount(collective) && (
+            <MenuLink isSub collective={collective} section={ALL_SECTIONS.FISCAL_HOSTING} />
+          )}
+        </React.Fragment>
+      )}
+    </React.Fragment>
+  );
+};
+
+OrganizationSettingsMenuLinks.propTypes = {
+  collective: PropTypes.object,
+  isAccountantOnly: PropTypes.bool,
+};
+
+const Menu = ({ collective, isAccountantOnly }) => {
+  const isHost = isHostAccount(collective);
+  const isIndividual = isIndividualAccount(collective);
+  const [expandedSection, setExpanded] = React.useState(null);
+
+  return (
+    <Container>
+      {/** Host dashboard */}
+      <MenuGroup if={isHost} mb={24}>
+        <MenuSectionHeader>
+          <FormattedMessage id="HostDashboard" defaultMessage="Host Dashboard" />
+        </MenuSectionHeader>
+        <MenuLink
+          setExpanded={setExpanded}
+          collective={collective}
+          section={HOST_SECTIONS.HOST_EXPENSES}
+          icon={<Receipt size={16} />}
+        />
+        <MenuLink
+          collective={collective}
+          section={HOST_SECTIONS.FINANCIAL_CONTRIBUTIONS}
+          icon={<Coin size={16} />}
+          setExpanded={setExpanded}
+          expanded={expandedSection === HOST_SECTIONS.FINANCIAL_CONTRIBUTIONS}
+        />
+
+        <MenuLink
+          collective={collective}
+          section={HOST_DASHBOARD_SECTIONS.PENDING_CONTRIBUTIONS}
+          icon={<Coin size={16} />}
+          if={!isAccountantOnly}
+        />
+
+        <MenuLink
+          collective={collective}
+          section={HOST_SECTIONS.PENDING_APPLICATIONS}
+          if={!isAccountantOnly}
+          icon={<Inbox size={16} />}
+          setExpanded={setExpanded}
+        >
+          Applications
```

Comment:
Removed this, this was a leftover in testing simpler names in the menu "Applications" instead of "Pending Applications", but that can be something to consider in a future PR.